src/api/typed.rs

```diff
@@ -120,6 +120,11 @@ where
         let req = self.api.log(name, lp)?;
         Ok(self.client.request_text(req).await?)
     }
+
+    pub async fn log_follow(&self, name: &str, lp: &LogParams) -> Result<impl Stream<Item = Result<Vec<u8>>>> {
+        let req = self.api.log(name, lp)?;
+        Ok(self.client.request_text_stream(req).await?)
+    }
```

Comment:
ah, now we have a separate log_follow method along with a parameter that changes the behaviour of logs. We should probably take the `follow` flag out of `LogParams` because the user has to decide what interface it wants anyway. Instead we can pass a `follow` bool or an enum into `self.api.log`.

Reply:
Cancel that ^^, see other two comments.

This function can work with and without `follow` set, technically, I think. So we should just rename this to `log_stream`.